app/controllers/project.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+class ProjectsController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /projects
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+		$pTitle		= "Projects";
+
+		$counter 	=	0;
+		$user 		=	User::find(Auth::id());
+		$projects 	=	$user->projects()->get();
+		$inProjects =  $user->inProjects()->orderBy('created_at', 'desc')->take(5)->get();
+		
+		return View::make('projects.index', compact(['projects','inProjects','counter','pTitle']));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "pTitle" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.137:9000/coding_rules#rule_key=php%3AS1192)